dev-tools/mage/kubernetes/kind.go

```diff
@@ -51,12 +51,30 @@ func (m *KindIntegrationTestStep) Setup(env map[string]string) error {
 	_, exists := env["KUBECONFIG"]
 	if exists {
 		// do nothing
-		return nil
+		if mg.Verbose() {
+			fmt.Println("KUBECONFIG: ", env["KUBECONFIG"])
+		}
+		if _, err := os.Stat(env["KUBECONFIG"]); err == nil {
+			return nil
+		} else if os.IsNotExist(err) {
+			if mg.Verbose() {
+				fmt.Println("KUBECONFIG file not found: ", env["KUBECONFIG"], err)
+			}
+		}
 	}
 	_, exists = env["KUBE_CONFIG"]
 	if exists {
 		// do nothing
-		return nil
+		if mg.Verbose() {
+			fmt.Println("KUBE_CONFIG: ", env["KUBE_CONFIG"])
+		}
+		if _, err := os.Stat(env["KUBE_CONFIG"]); err == nil {
+			return nil
+		} else if os.IsNotExist(err) {
+			if mg.Verbose() {
+				fmt.Println("KUBE_CONFIG file not found: ", env["KUBE_CONFIG"], err)
+			}
+		}
```

Comment:
Nit. This code is almost the same thing twice, could this be moved to a helper? Or at least to a loop?